spring-pulsar-dependencies/build.gradle

```diff
@@ -13,6 +13,7 @@ ext {
 	pulsarVersion = '2.10.2'
 	pulsarClientReactiveVersion = '0.1.0'
 	springBootVersion = '3.0.0'
+	springCloudStreamVersion = '4.0.0-SNAPSHOT'
```

Comment:
Everything else is locked on full GA versions currently. I think we should make them all snapshot or all GA for consistency. I think we locked onto GA just before we released. I understand the benefit of being on snapshots and I also understand the benefits of not being on snapshots :) 

However, we should choose the mode that the rest of the projects use. What does SK do?

Reply:
Ya, now that we have GA, we can use that for a while (`4.0.0` that is).